plugins/generator-1.16.5/forge-1.16.5/templates/tool.java.ftl

```diff
@@ -356,6 +356,35 @@ public class ${name}Item extends ${JavaModName}Elements.ModElement{
 			return ${data.enchantability};
 		}
 
+	}
+<#elseif data.toolType == "Fishing rod">
+    private static class ItemToolCustom extends FishingRodItem {
+
+		protected ItemToolCustom() {
+			super(new Item.Properties()
+				.group(${data.creativeTab})
+				.maxDamage(${data.usageCount})
+				<#if data.immuneToFire>
+				.isImmuneToFire()
+				</#if>
+			);
+		}
+
+		@Override
+		public boolean onBlockDestroyed(ItemStack stack, World worldIn, BlockState state, BlockPos pos, LivingEntity entityLiving) {

```

Comment:
Is there a particular reason why you added these, default fishing rods do not have these? Same question for 1.15.2.

Reply:
I pasted the code of Special tool and I forgot to remove this method.